src/main/java/it/pagopa/pn/bff/service/TosPrivacyService.java

```diff
@@ -26,6 +26,77 @@ public class TosPrivacyService {
     private final PnUserAttributesClientImpl pnUserAttributesClient;
     private final PnBffExceptionUtility pnBffExceptionUtility;
 
+
+    /**
+     * Accept or decline tos and privacy consents
+     *
+     * @param xPagopaPnUid    User Identifier
+     * @param xPagopaPnCxType Public Administration Type
+     * @param tosPrivacyBody  Body of the request containing the acceptance of the TOS and Privacy
+     * @return void
+     */
+//    public Mono<Void> acceptOrDeclinePgTosPrivacy(String xPagopaPnUid,
```

Comment:
remove the commented code but leave the javadocs

Reply:
done